controllers/managedocs_controller_test.go

```diff
@@ -669,6 +683,65 @@ var _ = Describe("ManagedOCS controller", func() {
 				utils.WaitForResource(k8sClient, ctx, promTemplate.DeepCopy(), timeout, interval)
 			})
 		})
+		When("there is no pagerduty secret in the cluster", func() {
+			It("should not create alertmanager config secret", func() {
+				// Verify that a pagerduty secret is not present
+				secret := pdSecretTemplate.DeepCopy()
+				Expect(k8sClient.Get(ctx, utils.GetResourceKey(secret), secret)).Should(
+					WithTransform(errors.IsNotFound, BeTrue()),
+				)
+
+				// Ensure, over a period of time, that the resources are not created
+				resList := []runtime.Object{
+					amConfigSecretTemplate.DeepCopy(),
+				}
+				utils.EnsureNoResources(k8sClient, ctx, resList, timeout, interval)
```

Comment:
No need to use the list version of the util

```suggestion
				utils.EnsureNoResource(k8sClient, ctx, amConfigSecretTemplate.DeepCopy(), timeout, interval)
```